solutions/enterprise-java/_common/solution.yml

```diff
@@ -16,8 +16,8 @@ primary_promo:
   - title: Red Hat JBoss EAP
     description: "An application platform that provides an innovative modular, cloud-ready architecture; powerful management and automation; and world class developer productivity."
     links:
-      - name: "Infographic: Accelerate App development w/ EAP"
-        url: 'https://www.redhat.com/cms/public/Jaen%20Swart_Devops%20with%20EAP%20and%20Openshift.pdf'
+      - name: "Get started with EAP 7"
+        url: '/products/eap/get-started/'
```

Comment:
This won't work in staging-pr, it produces: http://developers-pr.stage.redhat.com/products/eap/get-started/

I think you need the 'local_url' key, not 'url'. See elsewhere in this file.